File: BugTracker.API/Helpers/Counts.cs
namespace BugTracker.API.Helpers
{
    public class Counts
    {
        public int ActiveTickets { get; set; }
        public int ArchivedTickets { get; set; }
        public int TotalTickets { get; set; }
        public int PassedTickets { get; set; }
        public int ApprovedTickets { get; set; }
        public int ProjectActiveTickets { get; set; }   // Manager and Dev
        public int ProjectArchivedTickets { get; set; }     // Manager and Dev
        public int DevActiveTickets { get; set; }   // Dev
        public int DevArchivedTickets { get; set; }     // Dev


        public int ActiveProjects { get; set; }
        public int ArchivedProjects { get; set; }
        public int TotalProjects { get; set; }
        public int ProjectId { get; set; }  // Manager and Dev
        public string ProjectTitle { get; set; } // Manager and Dev

        public int TotalUsers { get; set; }
        public int FreeUsers { get; set; }
        public int BusyUsers { get; set; }
        public int ProjectUsers { get; set; }   // Manager and Dev

        public int Comments { get; set; }
        public int Projectcomments { get; set; }    // Manager and Dev
        public int DevComments { get; set; }    // Dev
    }
}

File: BugTracker-SPA/src/app/_models/Counts.ts
export interface Counts {

    // Admin
    totalTickets: number;
    activeTickets: number;
    archivedTickets: number;
    passedTickets: number;
    approvedTickets: number;

    totalProjects: number;
    activeProjects: number;
    archivedProjects: number;

    totalUsers: number;
    freeUsers: number;
    busyUsers: number;
    comments: number;


    // Manager and Developer
    projectId: number;
    projectTitle: string;

    projectActiveTickets: number;
    projectArchivedTickets: number;

    projectUsers: number;
    projectComments: number;


    // Developer
    devActiveTickets: number;
    devArchivedTickets: number;

    devComments: number;


    // All
    userName: string;
}
